client/components/LearnerViewDirective.js

```diff
@@ -35,8 +35,10 @@ tie.directive('learnerView', [function() {
                   </div>
                   <div ng-class="{'tie-step-line': $index < (questionIds.length - 1)}"></div>
                 </div>
+                <div ng-class="{'tie-step-line': $index < (questionIds.length - 1)}"></div>
               </div>
             </div>
+          </div>
```

Comment:
I thought you added this &lt;/div&gt;. My original comment was what was this terminating (because it didn't look like you added a new &lt;div&gt; above it). Basically, I think you terminated div class="tie-question-ui-inner" when you are not supposed to. Take a look at the original file and your earlier commit, and I think you will see that you added a lone &lt;/div&gt; there.